lib/modules/apostrophe-login/index.js

```diff
@@ -853,15 +886,23 @@ module.exports = {
         }
       }
 
-      return self.callAllAndEmit('loginAfterLogin', 'after', req, function(err) {
-        if (err) {
-          self.apos.utils.error(err);
-          return res.redirect('/');
-        }
+      var user = Object.assign({}, req.user, { lastLogin: new Date() });
```

Comment:
This needs to become a mongodb `$set` operation.

Reply:
Done